src/pages/app/searching_project_user/ProjectInfoPage.js

```diff
@@ -29,8 +29,9 @@ import IndividualForm from "../../../components/app/ProjectInfoPage/IndividualFo
 import * as projectActions from "../../../store/actions/searching-project-user/project/projectAction";
 import * as activityActions from "../../../store/actions/posting-project-user/activity/activity";
 
-const userType = window.localStorage.getItem("userType");
-console.log("User Type is:", userType);
+//const userId = JSON.parse(localStorage.getItem("userData")).userId;
+const userId = "1lrR6G5aoc0CuAaIrRN4";
```

Comment:
??? Static ??? @Chillisaucery

Reply:
I remember mentioning this issue somewhere... not GitHub issues, but still somewhere... Anw, since you fixed the Firestore I can fix this into dynamic now. Will fix it tomorrow, man, at this point I can barely remember what I have to do...